config_project_settings

```diff
@@ -0,0 +1,95 @@
+---
+title: ''
+author: "Jacob Kastl"
+date: "7/27/2020"
+output: html_document
+---
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+
+
+---
+title: "PACTA_analysis - New config file: project_settings"
+description: |
+  This post outlines the changes of the workflow in the PACTA_analysis repository. Everyone who works with this repository is encouraged to read this.
+author:
+  - name: Jacob Kastl
+    url: https://github.com/jacobvjk
+base_url: https://2degreesinvesting.github.io/
+date: 2020-07-27
+output: 
+  distill::distill_article:
+    toc: true
+    toc_depth: 3
+    self_contained: true
+categories:
+  - other
+  - programming
+twitter:
+  site: "@JacKastl"
+  creator: "@JacKastl"
+draft: true
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(
+  echo = TRUE,
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+**Target Audience:**
```

Comment:
Consider replacing `**Heading**` with `## Heading`. That will make your headings show up in the table of contents, which then becomes a nice overview of the article.